tests/test_blscache.py

```diff
@@ -1,11 +1,100 @@
-from chia_rs import G1Element, PrivateKey, AugSchemeMPL, G2Element, BLSCache
+from chia_rs import (
+    SpendBundle,
+    CoinSpend,
+    Program,
+    G1Element,
+    GTElement,
+    PrivateKey,
+    AugSchemeMPL,
+    G2Element,
+    BLSCache,
+    Coin,
+    ConsensusConstants,
+    validate_clvm_and_signature,
+)
+from chia_rs.sized_bytes import bytes32
+from chia_rs.sized_ints import uint8, uint16, uint32, uint64, uint128
 from typing import List
 from chia.util.hash import std_hash
 from chia.util.lru_cache import LRUCache
+from chia.types.blockchain_format.program import Program as ChiaProgram
 from chia.util import cached_bls as cached_bls_old
 import pytest
 
 
+DEFAULT_CONSTANTS = ConsensusConstants(
```

Comment:
it looks like this is only used by tests. I believe we already have a `TEST_CONSTANTS` that you could use. If that doesn't work, could you at least only define this when building with `#[cfg(test)]`?

Reply:
This is Python code - that said we may be able to export it as a binding for test purposes?